content/docs/setup/behavioral_changes.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Behavioral Changes in 1.1
+description: Major changes in behavior, both backward compatibility breaking and non, in Istio 1.1.
```

Comment:
This should probably be `Upgrade breaking changes in Istio 1.1.` given the direction from the document working group.